static/css/bureau-structure.less

```diff
@@ -87,14 +92,14 @@
         });
 
         &_name {
-            .respond-to-min(600, {
+            .respond-to-min(600px, {
                 .h4();
                 margin-bottom: unit(15px / @font-size, em);
                 text-align: center;
             });
         }
     }
-    
+
     &_role {
         padding: unit(@grid_gutter-width / 2, px);
         padding-right: unit(@grid_gutter-width, px);
```

Comment:
[`@grid_gutter-width`](https://github.com/cfpb/cf-grid/blob/gh-pages/src/cf-grid.less#L42) is already in pixels.